src/libraries/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1432,8 +1432,7 @@ public void BeginReceiveMessageFromV6BoundToSpecificV4_NotReceived()
         }
 
         [Fact]
-        [SkipOnPlatform(TestPlatforms.Linux | TestPlatforms.OSX | TestPlatforms.MacCatalyst | TestPlatforms.iOS | TestPlatforms.tvOS, "Expected behavior is different on Apple platforms and Linux")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/81946", TestPlatforms.Android)]
+        [SkipOnPlatform(TestPlatforms.Linux | TestPlatforms.Android | TestPlatforms.OSX | TestPlatforms.MacCatalyst | TestPlatforms.iOS | TestPlatforms.tvOS, "Expected behavior is different on Apple platforms, Linux and Android")]
```

Comment:
It only leaves Windows, right? It may be more readable to turn it into Windows specific test rather than adding more skips....